src/CommonDBTM.php

```diff
@@ -5332,9 +5339,7 @@ public static function listTemplates($itemtype, $target, $add = false)
 
         $request = [
             'FROM'   => $item::getTable(),
-            'WHERE'  => [
-                'is_template'  => 1,
-            ],
+            'WHERE'  => $item->getTemplateListCriteria(),
```

Comment:
You can use the `getSystemSQLCriteria()` instead.

```suggestion
            'WHERE'  => [
                'is_template' => 1,
            ] + static::getSystemSQLCriteria(),
```